changelog.md

```diff
@@ -0,0 +1,116 @@
+# Microsoft Orleans Changelog
+All notable end-user facing changes are documented in this file.
+
+### [vNext]
+*Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
+The idea is to track end-user facing changes as they occur.*
+
+- Sample change. We should start adding items here when we submit PRs
+
+
+
+### [v1.2.2]
+- Bugfix: Remote stacktrace is once again being included in the exception that bubbles up to the caller (bug introduced in 1.2.0). #1808
```

Comment:
Will these links to PR actually work, e.g. #1808 instead of https://github.com/dotnet/orleans/pull/1808?